docs/UserGuide.md

```diff
@@ -268,28 +268,19 @@ Format: `find [n/NAME] [s/SCHOOL] [y/YEAR]`
 * At least one of the optional fields must be provided.
 * The order of the optional fields do not matter. e.g `n/Hans s/River Valley` will match `s/River Valley n/Hans`
 * Only full words will be matched. e.g `han` will not match `hans`.
-* For the name, students with a name that matches any whole keyword specified for the name will be considered to match for the name.
-* For the school, students with a school that contains all keyword specified for the school will be considered to match for the school.
+* For the name, students with a name that matches any whole keyword specified will be matched.
+* For the school, students with a school that contains all keywords specified will be matched.
+* For the year, only students with the same year will be matched. (See below for more notes)
```

Comment:
Perhaps it'd be clearer if you phrased it as "For xxx search"
e.g "For name search", "For school search" and "For year search"
"with a name that" and "with a school that" could be further shortened to "whose name/school"

Reply:
Oops I left it out by accident. I meant more elaboration for how the year is parsed and interpreted by Reeve!